datasets/paws/paws.py

```diff
@@ -0,0 +1,209 @@
+
+NOTE: There might be some missing or wrong labels in the dataset and we have replaced them with -1.
+"""
+
+_HOMEPAGE = "https://github.com/google-research-datasets/paws"
+
+_LICENSE = 'The dataset may be freely used for any purpose, although acknowledgement of Google LLC ("Google") as the data source would be appreciated. The dataset is provided "AS IS" without any warranty, express or implied. Google disclaims all liability for any damages, direct or indirect, resulting from the use of the dataset.'
+
+_DATA_OPTIONS = [
+    "labeled_final",
+    "labeled_swap",
+    "unlabeled_final",
+]
+
+
+class PAWSConfig(datasets.BuilderConfig):
+    """BuilderConfig for PAWS."""
+
+    def __init__(self, **kwargs):
+        """Constructs a PAWSConfig.
+        Args:
+            **kwargs: keyword arguments forwarded to super.
+        """
+        super(PAWSConfig, self).__init__(version=datasets.Version("1.1.0", ""), **kwargs),
+
+
+class PAWS(datasets.GeneratorBasedBuilder):
+    """PAWS, a dataset for paraphrase identification"""
+
+    VERSION = datasets.Version("1.1.0")
+
+    BUILDER_CONFIGS = [
+        PAWSConfig(
+            name=config_name,
+            description=(f"This config contains samples of {config_name}."),
+        )
+        for config_name in _DATA_OPTIONS
+    ]
+
+    def _info(self):
+        features = datasets.Features(
+            {
+                "id": datasets.Value("int32"),
+                "sentence1": datasets.Value("string"),
+                "sentence2": datasets.Value("string"),
+                "label": datasets.features.ClassLabel(names=["0", "1"]),
+            }
+        )
+        return datasets.DatasetInfo(
+            # This is the description that will appear on the datasets page.
+            description=_DESCRIPTION,
+            # This defines the different columns of the dataset and their types
+            features=features,  # Here we define them above because they are different between the two configurations
+            # If there's a common (input, target) tuple from the features,
+            # specify them here. They'll be used if as_supervised=True in
+            # builder.as_dataset.
+            supervised_keys=None,
+            # Homepage of the dataset for documentation
+            homepage=_HOMEPAGE,
+            # License for the dataset if available
+            license=_LICENSE,
+            # Citation for the dataset
+            citation=_CITATION,
+        )
+
+    def _split_generators(self, dl_manager):
+        """Returns SplitGenerators."""
+
+        _DATA_URL = f"https://storage.googleapis.com/paws/english/paws_wiki_{self.config.name}.tar.gz"
+        data_dir = dl_manager.download_and_extract(_DATA_URL)
+
+        if self.config.name == "labeled_final":
+            _TRAIN_FILE_NAME = os.path.join(data_dir, "final/train.tsv")
+            _VAL_FILE_NAME = os.path.join(data_dir, "final/dev.tsv")
+            _TEST_FILE_NAME = os.path.join(data_dir, "final/test.tsv")
```

Comment:
```suggestion
            _TRAIN_FILE_NAME = os.path.join(data_dir, "final", "train.tsv")
            _VAL_FILE_NAME = os.path.join(data_dir, "final", "dev.tsv")
            _TEST_FILE_NAME = os.path.join(data_dir, "final", "test.tsv")
```

to make it work for windows users